src/assets/translations/fr/main.json

```
@@ -1871,7 +1886,8 @@
   },
   "ledgerOpenApp": {
     "title": "Ouvrir l'app %{appName}",
-    "description": "Pour continuer, vous devrez ouvrir l'application %{appName} sur votre appareil Ledger."
+    "description": "Pour continuer, vous devrez ouvrir l'application %{appName} sur votre appareil Ledger.",
```

While at it, feels more natural IMO!
```suggestion
    "description": "Pour continuer, vous devrez ouvrir l'application %{appName} sur votre Ledger.",
```

---

We've used "appareil Ledger" in few of other places to designate Ledger and the English copies say "Ledger device" when they could also just say "Ledger". I'm for sticking to literal translations/consistency between copies so if we're going to change this it should be changed everywhere, so let's skip this one for now.